packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -228,6 +229,18 @@ class DropdownMenu<T> extends StatefulWidget {
   /// Defaults to null. If null, only the text field is updated.
   final ValueChanged<T?>? onSelected;
 
+  /// Determine if the dropdown button requests focus and the on-screen virtual
+  /// keyboard is shown in response to a touch event.
+  ///
+  /// By default, if the app is running on mobile platforms, the text field
+  /// will not request focus when the menu is opened, and the virtual keyboard
+  /// will not show up. If the app is running on desktop platforms, the text field
+  /// requests focus when the menu is opened.
+  ///
+  /// Defaults to null. Setting this to false is useful when the options are limited and searching
```

Comment:
Setting this field to true or false, rather than allowing the implementation to choose based on the platform, can be useful for applications that want to override the default behavior.